client/src/pages/Login/DemoButton.tsx

```diff
@@ -1,16 +1,12 @@
 import Button from '@mui/material/Button';
 import Box from '@mui/material/Box';
 import useStyles from './LoginForm/useStyles';
-import { useSnackBar } from '../../context/useSnackbarContext';
-import login from '../../helpers/APICalls/login';
-import { useAuth } from '../../context/useAuthContext';
 
 interface DemoButtonProps {
-  demoEmail: string;
-  demoPassword: string;
+  handleClick: VoidFunction;
 }
 
-const DemoButton: React.FC<DemoButtonProps> = ({ demoEmail, demoPassword }) => {
+function DemoButton(props: DemoButtonProps): JSX.Element {
```

Comment:
Could we go back to the arrow function style? Just for consistency

Reply:
Could we also move this to the components since it's used in Login and Signup too. Would make more sense to have it there. 😄

Reply:
Changes in latest version:
- Made the function an arrow function
- Moved DemoButton to the components section - thanks for pointing this out!